File: tests/testdata/codegen/inline-invokes-pp/nodejs/inline-invokes.ts
import * as codeinfra from "@codeinfra/codeinfra";
import * as aws from "@codeinfra/aws";

const webSecurityGroup = new aws.ec2.SecurityGroup("webSecurityGroup", {vpcId: aws.ec2.getVpc({
    "default": true,
}).then(invoke => invoke.id)});

File: tests/testdata/codegen/inline-invokes-pp/dotnet/inline-invokes.cs
using System.Collections.Generic;
using System.Linq;
using Codeinfra;
using Aws = Codeinfra.Aws;

return await Deployment.RunAsync(() => 
{
    var webSecurityGroup = new Aws.Ec2.SecurityGroup("webSecurityGroup", new()
    {
        VpcId = Aws.Ec2.GetVpc.Invoke(new()
        {
            Default = true,
        }).Apply(invoke => invoke.Id),
    });

});

